### DISCUSSION TITLE
DjangoFilterConnectionField, Apollo, Vue, Relay.Node.....................

### DISCUSSION THREAD

**Discussion by exisz** [graphql-python/graphene-django#1168] [Q&A]
As per getting started documentations, 

It seems the official recommended way to set up the project is with Relay.Node, 

Integration with django-filters **DjangoFilterConnectionField** rely on Relay integration.

Further, I need below code:
```python
class Meta:
    interfaces = (Node,)
```

I am planning to use VueJS / Apollo. Does this mean I cannot go Apollo but have to use Relay, or does it not matter? What is the recommended stack?

**Comment by zbyte64** [comment]
Doesn't matter. Apollo speaks graphql, relay is just extra graphql batteries. graphene-django does not recommend any stack over another, we just make it easier to use graphql with django.
